pvlib/modelchain.py

```diff
@@ -260,6 +263,27 @@ def get_orientation(strategy, **kwargs):
     return surface_tilt, surface_azimuth
 
 
+@dataclass
+class ModelChainResult:
+    T = TypeVar('T')
+    PerArray = Union[T, Tuple[T, ...]]
+    # system-level information
+    solar_position: pd.DataFrame = field(default=None)
+    airmass: pd.DataFrame = field(default=None)
+    ac: pd.Series = field(default=None)
+    # per DC array information
+    tracking: Optional[pd.DataFrame] = field(default=None)
+    total_irrad: Optional[PerArray[pd.DataFrame]] = field(default=None)
+    aoi: Optional[PerArray[pd.Series]] = field(default=None)
+    aoi_modifier: Optional[PerArray[pd.Series]] = field(default=None)
+    spectral_modifier: Optional[PerArray[pd.Series]] = field(default=None)
+    cell_temperature: Optional[PerArray[pd.Series]] = field(default=None)
+    effective_irradiance: Optional[PerArray[pd.Series]] = field(default=None)
+    dc: Optional[PerArray[pd.Series]] = field(default=None)
+    array_ac: Optional[PerArray[pd.Series]] = field(default=None)
```

Comment:
This is unused, should probably be removed. @cwhanse do you recall what this was supposed to be for?

Reply:
`array_ac` was added before working out how to combine multiple Arrays. OK to remove.